spread.yaml

```diff
@@ -75,3 +75,4 @@ path:
 
 exclude:
     - .git
+    - cmake-build-*
```

Comment:
Unsure why this…

Reply:
Oh, whoops. That's just CLion's default build directories; excluding them makes spread take about a minute less time compressing and transferring the source :wink:

I didn't mean to push that change, but maybe we want it anyway? Other people use CLion, too.